src/commands/service/create.ts

```diff
@@ -89,6 +89,21 @@ export const createCommandDecorator = (command: commander.Command): void => {
       "Kubernetes service port which this service is exposed with; will be used to configure Traefik2 IngressRoutes",
       "80"
     )
+    .option(
+      "--path-prefix <path-prefix>",
+      "The path prefix for ingress route; will be used to configure Traefik2 IngressRoutes. If omitted, then the service name will used.",
+      ""
+    )
+    .option(
+      "--path-prefix-version <path-prefix-version>",
+      "Version to be used in the path prefix; will be used to configure Traefik2 IngressRoutes. ie. 'v1' will result in a path prefix of '/v1/servicename",
+      ""
+    )
+    .option(
+      "--backend <backend>",
```

Comment:
maybe something like --k8s-backend

Reply:
we currently have `--k8s-service-port` to specify the port for it. perhaps `--k8s-service-name`